<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectNewAvatarWindow</class>
 <widget class="QWidget" name="SelectNewAvatarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>更换头像</string>
  </property>
  <widget class="LabelPlus" name="lbl_new_avator">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>91</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>45</y>
     <width>170</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>支持jpg,png格式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_select_avatar_file">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>45</y>
     <width>170</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上传本地照片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_yes">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>480</y>
     <width>72</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>72</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>6</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>更换头像</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelPlus</class>
   <extends>QLabel</extends>
   <header>LabelPlus.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
